.github/workflows/tests-labs.yml

```diff
@@ -67,5 +67,5 @@ jobs:
       python_version: '3.10'
       pytest_test_directory: pennylane/labs/tests
       additional_pip_packages: |
-        geometric mpi4py h5py
+        geometric mpi4py h5py optax basis_set_exchange pyscf
```

Comment:
@ddhawan11 and @mlxd for visibility, I had to add `basis_set_exchange` and `pyscf` here as I got errors from unrelated tests in `pennylane/labs/tests/vibrational/` 

[missing basis_set_exchange](https://github.com/PennyLaneAI/pennylane/actions/runs/12371319193/job/34527197398?pr=6446)
[missing pyscf](https://github.com/PennyLaneAI/pennylane/actions/runs/12376160686/job/34542666894?pr=6446)

It would also be good to understand how these got in in the first place without raising these errors in CI before 🤔

Reply:
Additionally now getting [these errors](https://github.com/PennyLaneAI/pennylane/actions/runs/12389864379/job/34583781023?pr=6446) 😿 

```pycon
 =========================== short test summary info ============================
FAILED pennylane/labs/tests/vibrational/test_pes_generator.py::test_twomode_pes[sym0-geom0-freqs0-vectors0-H2S.hdf5] - KeyError: "Unable to synchronously open object (object 'V1_PES' doesn't exist)"
FAILED pennylane/labs/tests/vibrational/test_pes_generator.py::test_threemode_pes[sym0-geom0-freqs0-vectors0-H2S.hdf5] - KeyError: "Unable to synchronously open object (object 'V1_PES' doesn't exist)"
======= 2 failed, 692 passed, 1 skipped, 9 warnings in 129.52s (0:02:09) =======
```

Reply:
I am wondering if this file got overwritten in one of the later PRs. Let me look into it. 
We should definitely try to figure out how these got past CI. Could it be that the lab tests didn't run in the CI at all?

Reply:
Seems like this file got overwritten here: https://github.com/PennyLaneAI/pennylane/pull/6560/files. @austingmhuang and I will open a quick PR to fix this.
I think it is important we figure out why these things passed the CI though, so that it doesn't happen again.

Reply:
The above mentioned PR changes things in the labs folder only, it still passed those tests. Moreover, the PR that added those tests changed things in the labs folder as well.

Reply:
I have opened a small PR to fix these issues https://github.com/PennyLaneAI/pennylane/pull/6730

Reply:
Fix seems to have worked, thanks a lot for the quick turnaround, awesome to see! :)